src/pages/costModels/components/inputs/selector.tsx

```diff
@@ -49,24 +88,25 @@ const SelectorBase: React.FunctionComponent<SelectorProps> = ({
       helperTextInvalid={helpText !== null && typeof helpText === 'object' ? intl.formatMessage(helpText) : helpText}
       validated={isInvalid ? 'error' : 'default'}
     >
-      <FormSelect
-        isRequired={isRequired}
-        isDisabled={isDisabled}
-        value={value}
-        onChange={onChange}
-        aria-label={ariaLabel}
+      <Select
         id={id}
```

Comment:
I don't see the ID output in the page. Might be related to the test issues?

<img width="1167" alt="Screen Shot 2022-03-29 at 9 10 58 AM" src="https://user-images.githubusercontent.com/17481322/160618777-f5aab2be-5aeb-4852-a1f8-610d04fbf867.png">

From stage-beta, I can see the ID "metric"

<img width="1162" alt="Screen Shot 2022-03-29 at 9 11 38 AM" src="https://user-images.githubusercontent.com/17481322/160618946-a1c41321-c868-4ac3-b061-6affe8ce6b99.png">

Reply:
it seems the ID provided will show up only on the dropdown menu (once opened)